ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -236,7 +236,9 @@ function prepare_next_steps() {
       "${dir}/auth/kubeconfig" \
       "${dir}/auth/kubeadmin-password" \
       "${dir}/metadata.json"
-
+  if [[ "${CLUSTER_TYPE}" == "vsphere"* ]] && [[ -f ${SHARED_DIR}/template.yaml.patch ]]; then
```

Comment:
`ipi-conf-vsphere-customized-resource` also used `template.yaml.patch`, need to update that step.